src/modapi/moditem.h

```diff
@@ -72,4 +73,56 @@ struct CModAPI_ModItem_LineStyle
 	int m_AnimationSpeed;
 };
 
+enum ESkinModifier
```

Comment:
The SKIN_ enums are used to determine whether a complete skin should be changed to another complete (yet existing vanilla) skin, or if a single/multiple part/s should be changed.